tests/test_usm_type.py

```diff
@@ -85,6 +85,29 @@ def test_coerced_usm_types_remainder(usm_type_x, usm_type_y):
 
     z = x % y
 
+    # inplace remainder
+    z %= y
+    z %= 5
+
+    assert x.usm_type == usm_type_x
+    assert y.usm_type == usm_type_y
+    assert z.usm_type == du.get_coerced_usm_type([usm_type_x, usm_type_y])
+
+
+@pytest.mark.parametrize("usm_type_x", list_of_usm_types, ids=list_of_usm_types)
+@pytest.mark.parametrize("usm_type_y", list_of_usm_types, ids=list_of_usm_types)
+def test_coerced_usm_types_floor_divide(usm_type_x, usm_type_y):
+    x = dp.arange(100, usm_type=usm_type_x).reshape(10, 10)
+    y = dp.arange(100, usm_type=usm_type_y).reshape(10, 10)
+    x = x + 1.5
+    y = y.T + 0.5
+
+    z = x // y
```

Comment:
Only one more comment is that it would be good to cover `dpnp.ndarray.__rfloordiv__()` here:
```suggestion
    z = 3.4 // x
    z = y // 2.7
    z = x // y
```